### PR TITLE
Fix building tools again

### PR DESCRIPTION
**Pull request by kant2002**
I miss that incorrectly missing files.
Also since common files used everywhere, TargetName cannot be set in shared files

### CODE REVIEW

File: VisualStudio/common.props [hunk 1]
```diff
@@ -5,7 +5,6 @@
   <PropertyGroup>
     <OutDir>$(MSBuildThisFileDirectory)..\build\$(PlatformTarget)\$(Configuration)\</OutDir>
     <IntDir>$(MSBuildThisFileDirectory)..\build\$(PlatformTarget)\$(Configuration)\$(ProjectName)\</IntDir>
-    <TargetName>fheroes2</TargetName>
```
Comment (ihhub) [inline VisualStudio/common.props:8]: Hi @kant2002 , could you please explain about this change?

**Comment by ihhub** [comment]
Hi @kant2002 ,

Is it possible to make VS2019 version as well? If yes then we can modify `.github/vs2019_pull_request.yml` to include tools compilation:
```
    - name: compile fheroes2
      run: MSBuild.exe fheroes2-vs2019.vcxproj /property:Configuration=${{ matrix.config-type }} /property:Platform=${{ matrix.platform-type }}
    - name: compile tools
      run: |
        cd src/tools
        MSBuild.exe 82m2wav.vcxproj /property:Configuration=${{ matrix.config-type }} /property:Platform=${{ matrix.platform-type }}
        MSBuild.exe bin2txt.vcxproj /property:Configuration=${{ matrix.config-type }} /property:Platform=${{ matrix.platform-type }}
        MSBuild.exe extractor.vcxproj /property:Configuration=${{ matrix.config-type }} /property:Platform=${{ matrix.platform-type }}
        MSBuild.exe icn2img.vcxproj /property:Configuration=${{ matrix.config-type }} /property:Platform=${{ matrix.platform-type }}
        MSBuild.exe til2img.vcxproj /property:Configuration=${{ matrix.config-type }} /property:Platform=${{ matrix.platform-type }}
        MSBuild.exe xmi2mid.vcxproj /property:Configuration=${{ matrix.config-type }} /property:Platform=${{ matrix.platform-type }}
```

Also the only difference between 2015 version and 2019 version is WindowsPlatformVersion and PlatformToolset. Do you think it's possible to make some common code based on only these differences?

File: VisualStudio/common.props (hunk 1 above)
Comment (kant2002) [reply VisualStudio/common.props:8]: This override executable name and lock .ilk file in the VS if you use more then one project in the solution. And I use solution where I compile all projects.

**Comment by ihhub** [comment]
@kant2002 , thank you very much for this fix!

**Pull request merged**